BootloaderCommonPkg/Include/Library/TpmLib.h

```diff
@@ -116,6 +116,24 @@ TpmExtendStageHash (
   IN       UINT8            ComponentType
   );
 
+
+/**
+  Extend  hash provided in PCR 0 with EV_POST_CODE event type.
+
+  @param[in] Digest                    Hash Digest to extend
+  @param[in] HashAlg                   Tcg Alg to extend
+
+  @retval RETURN_SUCCESS               Operation completed successfully.
+  @retval RETURN_INVALID_PARAMETER     Imvalid Digest  buf
+
+  @retval Others                       Unable to extend stage hash.
+**/
+RETURN_STATUS
+TpmExtendHash (

```

Comment:
How is this method different from TpmExtendStageHash?
Can we use a better name to imply the usage?

Reply:
In TPMExtendStageHash -  Hash would be retrieved from Hash Store based on component ID.
In TPMExtendHash - it would extend hash provided by calling function.